### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by Kha**

### CODE REVIEW

File: 06_Interacting_with_Lean.org [hunk 1]
```diff
@@ -495,7 +500,7 @@ with the extension =.olean=, for "object Lean".
 
 Assuming the directory that contains Lean is in your system path, you
 can run lean on a file =foo.lean= from a system command line by typing
-=lean foo=. If =foo= imports other files, by default Lean looks for
+=lean foo.lean=. If =foo= imports other files, by default Lean looks for
```
Comment (Kha) [inline 06_Interacting_with_Lean.org:503]: Perhaps even change every `foo` to `foo.lean`.

Comment (Kha) [reply 06_Interacting_with_Lean.org:503]: I'm a bit confused about this one, did it work without the file extension at some point?

Comment (avigad) [reply 06_Interacting_with_Lean.org:503]: I don't remember. I almost never used it from the command line...

**Comment by avigad** [comment]
This was merged with dd000a6.

**Pull request closed without merging**